CMakeLists.txt

```diff
@@ -50,14 +59,21 @@ if(NOT CMAKE_BUILD_TYPE AND NOT CMAKE_CONFIGURATION_TYPES)
 endif()
 
 include(GoogleBenchmark)
+if(flamegraph)
+  # Check if perf is available in OS:
+  find_program(PERF_EXECUTABLE perf)
+  if(NOT PERF_EXECUTABLE)
+    message(WARNING "Perf is not available in your system, please install it.")
+    set(flamegraph OFF CACHE BOOL "")
+  endif()
+  include(FlameGraph)
```

Comment:
What happens if `perf` is not found. Can we still create flamegraphs? Do we need to `include(FlameGraph)`?